pypapi/papi.py

```diff
@@ -72,3 +72,25 @@ def ipc():
             ffi.unpack(ins, 1)[0],
             ffi.unpack(ipc_, 1)[0]
             )
+
+
+def epc():
+    """Gets (named) events per cycle, real and processor time, reference and
+    core cycles
+    """
+    rtime = ffi.new("float*", 0)
+    ptime = ffi.new("float*", 0)
+    ref = ffi.new("long long*", 0)
+    core = ffi.new("long long*", 0)
+    epc_ = ffi.new("float*", 0)
+
+    rcode = lib.PAPI_epc(rtime, ptime, ref, core, epc_)
```

Comment:
This call does not match the function signature in `papi.h`:
* the `int event` param is missing
* the `long long *evt` param is missing